x/asset/keeper/keeper.go

```diff
@@ -0,0 +1,63 @@
+package keeper
+
+import (
+	"cosmossdk.io/log"
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/provenance-io/provenance/x/asset/types"
+	ledgerkeeper "github.com/provenance-io/provenance/x/ledger/keeper"
+	markerkeeper "github.com/provenance-io/provenance/x/marker/keeper"
+)
+
+// Keeper of the asset store
+type Keeper struct {
+	cdc            codec.BinaryCodec
+	storeKey       storetypes.StoreKey
+	nftKeeper      types.NFTKeeper
+	router         baseapp.MessageRouter
+	ledgerKeeper   ledgerkeeper.BaseKeeper
+	registryKeeper types.BaseRegistryKeeper
+	markerKeeper   markerkeeper.Keeper
```

Comment:
suggestion (blocking): The `ledgerKeeper` and `markerKeeper` need to be interfaces defined in asset types (same as the `NFTKeeper` in here).

Using interfaces makes it possible for two keepers to use each other. Without interfaces, that results in an import loop compilation issue. Using an interface also allows for mocking, which should make unit tests easier and more accurate.